app/assets/images/caret_down.svg

```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" data-prefix="fas" data-icon="caret-down" class="svg-inline--fa fa-caret-down fa-w-10" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="white" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"></path></svg>
```

Comment:
Directly downloaded svg from Font Awesome, I can't find a way to import SVGs to SCSS files

Reply:
Oh interesting. Does referencing the SVG path in `url()` not do anything?

Reply:
There's no way to get the path for the SVG at compile time that I found :/

Reply:
Oohhh I see. Well. Not the end of the world. It might actually be better to keep the SVG directly in the markup in case we want to apply transitions to it in the future.

Reply:
Good point! I'll poke around try that instead.

Reply:
@nshki Updated!